src/main/app/case/CaseApi.ts

```diff
@@ -33,12 +33,8 @@ export class CaseApi {
    * @param formData
    * @returns
    */
-  public async getOrCreateCaseNew(
-    req: AppRequest,
-    userDetails: UserDetails,
-    formData: Partial<Case>
-  ): Promise<CaseWithId> {
-    return this.createCaseNew(req, userDetails, formData);
+  public async getOrCreateCaseNew(req: AppRequest, userDetails: UserDetails): Promise<CaseWithId> {
```

Comment:
do we actually need this method? can't we directly call createCaseNew from controller?